receiver/splunksearchapireceiver/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	go.opentelemetry.io/collector/extension/experimental/storage v0.113.0
 	go.opentelemetry.io/collector/pdata v1.19.0
 	go.opentelemetry.io/collector/receiver v0.113.0
+	go.opentelemetry.io/collector/receiver/receivertest v0.113.0
```

Comment:
Did this not rmake any go.sum changes? I'd imagine it would. Would you mind re-running `go mod tidy` to just double check?

Reply:
Apparently not 🤷‍♂️